docs/application/dotnet/guides/connectivity/internet-download.md

```diff
@@ -9,7 +9,10 @@ You can use the following Internet and content download features in your .NET ap
 
     You can get information about the connected network, and set up and configure network and socket connections. You can also track connection information, such as the size of the sent and received data.
 
+-   [WebView](webview.md)
+
+    The WebView functionality enables you to access web pages and other web content in your application. You can also use various other features for web browsing, such as loading, displaying web pages, and navigating through the browsing history.
 
```

Comment:
@annie-samsung As I asked with [this comment](https://github.com/Samsung/tizen-docs/pull/742#discussion_r293214475), I also think using similar sentence is better.

> mijin85cho 12 days ago  Member
> @nawab-samsung Please change this list into sentence form. In Tizen Document, list form is not used in this way at the cover pages. Please **only check the grammar** of below sentence, and **don't try to polish** the sentence.
>
>"You can access Web pages and Web content in your application. You can also use various features for Web browsing, such as loading and displaying Web pages and navigating through the browsing history."

Another issue. You separated "loading", "displaying web pages", and "navigating browser history", using "A, B, and C" rule.
However isn't the loading and displaying is for web pages, and navigating is for browser history?

I mean it looks like to me that this sentence is "such as A and B" form, while
   - A is "loading and displaying Web pages" and
   - B is "navigating through the browsing history"

Reply:
@mijin85cho 
Agreed, since in native guide it is written the same way.

You can also use various other features for Web browsing, such as loading and displaying web pages, and navigating through the browsing history.

Reply:
@younghajung, 
I and @mijin85cho discussed and agreed on the same. Kindly make the following change:
___________________________

You can access web pages and web content in your application using the WebView functionality. You can also use various other features for Web browsing, such as loading and displaying web pages, and navigating through the browsing history.

Reply:
Done.